File: dataset/python2java/python/EQUILIBRIUM_INDEX_OF_AN_ARRAY.py
def f_gold ( arr , n) :
    leftsum = 0
    rightsum = 0
    for i in range ( n ) :
        leftsum = 0
        rightsum = 0
        for j in range ( i ) :
            leftsum += arr [ j ]
        for j in range ( i + 1 , n ) :
            rightsum += arr [ j ]
        if leftsum == rightsum :
            return i
    return - 1

File: result/translate/chatgpt/python2java/java/EQUILIBRIUM_INDEX_OF_AN_ARRAY.java
public static int f_gold(int[] arr, int n) {
    int leftsum = 0;
    int rightsum = 0;
    for (int i = 0; i < n; i++) {
        leftsum = 0;
        rightsum = 0;
        for (int j = 0; j < i; j++) {
            leftsum += arr[j];
        }
        for (int j = i + 1; j < n; j++) {
            rightsum += arr[j];
        }
        if (leftsum == rightsum) {
            return i;
        }
    }
    return -1;
}